### PR TITLE
Remove Package, AssetImporter, and other old resources stuff

### PR DESCRIPTION
**Pull request by Capital-Asterisk**
Depends on #179

:hocho:

### CODE REVIEW

File: src/adera/ShipResources.h [hunk 1]
```diff
@@ -141,7 +140,7 @@ struct ShipResourceType
 
 struct ShipResource
 {
-    osp::DependRes<ShipResourceType> m_type;
+    //osp::DependRes<ShipResourceType> m_type;
```
Comment (jonesmz) [inline src/adera/ShipResources.h:143]: imho, just erase the line

File: src/osp/Resource/blueprints.h [hunk 2]
```diff
@@ -26,15 +26,14 @@
...
-#include "PrototypePart.h"
+#include "resources.h"
 
 namespace osp
 {
 
 struct BlueprintMachine
 {
-    NodeMap_t m_config;
+    //NodeMap_t m_config;
```
Comment (jonesmz) [inline src/osp/Resource/blueprints.h:36]: remove

File: src/test_application/VehicleBuilder.cpp [hunk 3]
```diff
@@ -24,6 +24,8 @@
  */
 #include "VehicleBuilder.h"
 
+#if 0
```
Comment (jonesmz) [inline src/test_application/VehicleBuilder.cpp:27]: you've basically deleted the whole file?

**Review by jonesmz — APPROVED** [review]

File: src/adera/ShipResources.h (hunk 1 above)
Comment (Capital-Asterisk) [reply src/adera/ShipResources.h:143]: Something will eventually replace this, though I don't fully remember how the ShipResource stuff works. I intentionally left this not deleted so its original intention can still be worked out.

File: src/osp/Resource/blueprints.h (hunk 2 above)
Comment (Capital-Asterisk) [reply src/osp/Resource/blueprints.h:36]: There's a chance that NodeMap_t will be typedefed to something else eventually, though I'm not sure what yet. Similar case to other comment above

File: src/test_application/VehicleBuilder.cpp (hunk 3 above)
Comment (Capital-Asterisk) [reply src/test_application/VehicleBuilder.cpp:27]: yep. Most of the functions should remain the same once vehicles are re-implemented.

**Pull request merged**